languages/tree-sitter-stack-graphs-javascript/src/stack-graphs.tsg

```diff
@@ -3827,83 +3881,6 @@ inherit .containing_class_value
 
 }
 
-(nested_identifier)@nested_identifier {
-  node @nested_identifier.before_scope
-  node @nested_identifier.after_scope
-  node @nested_identifier.value
-
-  edge @nested_identifier.after_scope -> @nested_identifier.before_scope
-}
-
-(nested_identifier
-  (identifier)@first_part
-  (identifier)@second_part)@nested_identifier
-{
-  node guard_member
-
-  attr (@first_part.value) node_reference = @first_part
-  attr (@second_part.value) node_reference = @second_part
-  attr (guard_member) symbol_reference = "GUARD:MEMBER"
-
-  edge @first_part.value -> @nested_identifier.before_scope
-  edge guard_member -> @first_part.value
-  edge @second_part.value -> guard_member
-  edge @nested_identifier.value -> @second_part.value
-}
-
-(nested_identifier
-  (nested_identifier)@first_part
-  (identifier)@second_part)@nested_identifier
-{
-  node guard_member
-
-  attr (@second_part.value) node_reference = @second_part
-  attr (guard_member) symbol_reference = "GUARD:MEMBER"
-
-  edge @first_part.before_scope -> @nested_identifier.before_scope
-  edge guard_member -> @first_part.value
-  edge @second_part.value -> guard_member
-  edge @nested_identifier.value -> @second_part.value
-}
```

Comment:
These separate rules are obsolete now that `nested_identifier` is renamed to `member_expression.`